step-ca/certificate-authority-server-production.mdx

```diff
@@ -163,8 +163,7 @@ You can [configure certificate lifetimes](./configuration.mdx#basic-configuratio
 
 <Alert severity="info">
   <div>
-    Short-lived certificates are not a replacement for active revocation using certificate revocation lists (CRL) or Online Certificate Status Protocol (OCSP).
-	Automatic active revocation is not available in <Code>step-ca</Code>, but you can manually manage a CRL for your CA certificates. See below.
+    Short-lived certificates are not a replacement for active revocation using certificate revocation lists (CRL) or Online Certificate Status Protocol (OCSP). They offer an alternative that's easier to deploy and maintain.
```

Comment:
Strictly not indeed, but maybe a little bit of nuance to it: "are not a full/complete replacement"?